cloud-infrastructure/storage/block-storage/asset/trim-showcase.md

```diff
@@ -0,0 +1,199 @@
+# License
+
+Copyright (c) 2023 Oracle and/or its affiliates.
+
+Licensed under the Universal Permissive License (UPL), Version 1.0.
+
+See [LICENSE](https://github.com/oracle-devrel/technology-engineering/blob/folder-structure/LICENSE) for more details.
+
+
+# Showcase TRIM
+
+## Prepare your environment
+- [Oracle Cloud Infrastructure CLI Command Reference](https://docs.oracle.com/en-us/iaas/tools/oci-cli/latest/oci_cli_docs/index.html)
+- set up your individual "operate" instance
+	- An Oracle enterprise Linux server with Command Line Interface (*see [Set up your Server with Resilience by default using CLI](https://gitlab.com/hmielimo/cloud-resilience-by-default/#set-up-your-server-with-resilience-by-default-using-cli)  for details*)
+	- create and attach block volume: e.g. trim-test (*200GB*)
+
+## Format and mount block volume
+~~~
+DEVICEoPATH=/dev/oracleoci/oraclevdb
+DATAoDIR=/mnt/MyBlockVolume
+sudo mkdir /mnt/MyBlockVolume
+sudo mkfs -t ext4 ${DEVICEoPATH}
+sudo mount ${DEVICEoPATH} ${DATAoDIR}
+sudo chown -R opc:opc ${DATAoDIR}
+ls -lah ${DATAoDIR}
+sudo mount | grep ${DATAoDIR}
+~~~
+
+## Update fstab
+~~~
+sudo vi /etc/fstab
+/dev/oracleoci/oraclevdb  /mnt/MyBlockVolume    ext4    defaults,_netdev,noatime  0  2
+~~~
+
+## TRIM OFF showcase
+
+
+### Set needed variables (*please adjust variables to your needs before past into your terminal*)
+~~~
+# ---------------------------------------------------------------------------------------------------------------------------------------------
+# CUSTOMER SPECIFIC VALUES - please update appropriate
+# ---------------------------------------------------------------------------------------------------------------------------------------------
+export TENENCY_OCID=<your_tenency_ocid>
+export USER_OCID=<your_user_ocid>
+export COMPARTMENT_OCID=<your_compartment_ocid>
+export FRANKFURT_SUBNET_OCID=<your_frankfurt_subnet_ocid>
```

Comment:
Why FRANKFURT? Can this be region neutral?

Reply:
done